src/app/screens/PeerlyScreens/HomeScreen/index.tsx

```diff
@@ -75,9 +84,12 @@ const HomeScreen = () => {
       <View style={styles.header}>
         <Text style={styles.title}>Peerly</Text>
         <View style={styles.userScore}>
-          <Text style={styles.scoreText}>2000</Text>
+          <Text style={styles.scoreText}>{profileDetails?.total_points}</Text>
```

Comment:
please add a placeholder to show when total_points is undefined.

Reply:
Done